src/types.ts

```diff
@@ -0,0 +1,56 @@
+export interface Props {
+    data: QueriedData
+}
+interface QueriedData {
+    count: number,
+    next: string,
+    previous: string | null,
+    results: BeneficiaryData[]
+}
+
+export interface BeneficiaryData {
```

Comment:
lets call this just `Beneficiary` and not `BeneficiaryData` since we know it is a beneficiary object which consists of that data.